src/bijectors/stacked.jl

```diff
@@ -47,7 +48,7 @@ isclosedform(b::Stacked) = all(isclosedform, b.bs)
 
 isinvertible(b::Stacked) = all(isinvertible, b.bs)
 
-stack(bs...) = Stacked(bs)
+@deprecate stack(bs...) Stacked(bs...) false
```

Comment:
I'm starting to wonder - is this deprecation useful at all if `Bijectors.stack` is not exported? Is that helpful for DynamicPPL?

Reply:
Also `stack` is removed from the tests :shrug: So maybe just remove it completely?
```suggestion
```